README.md

```diff
@@ -1,2 +1,13 @@
-# UI-design
-User-interface design for WeConnect which is a platform that brings businesses and individuals together.
+WeConnect
+Andela Project Challage
```

Comment:
I think you can give more context on the weconnect short description rather than it's an Andela Project Challenge